java-checks/src/test/files/checks/security/SecureXmlTransformerCheck.java

```diff
@@ -0,0 +1,88 @@
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.XMLConstants;
+
+class A {
```

Comment:
would it be useful to add some `This implementation will not check` FN cases, just to be clear from the tests what the rule does not cover?

Reply:
Added in eb6a439ed.